### PR TITLE
added docs for Xiaomi CGG1 sensor

### PR DESCRIPTION
**Pull request by Alex9779**
## Description:

**Related issue (if applicable):** fixes none

**Pull request in [esphome](https://github.com/esphome/esphome) with YAML changes (if applicable):** esphome/esphome#746

## Checklist:

  - [x] Branch: `next` is for changes and new documentation that will go public with the next ESPHome release. Fixes, changes and adjustments for the current release should be created against `current`.

### CODE REVIEW

File: components/sensor/xiaomi_cgg1.rst [hunk 1]
```diff
@@ -0,0 +1,96 @@
+Xiaomi CGG1 BLE Sensor
+=======================
```
Comment (OttoWinter) [inline components/sensor/xiaomi_cgg1.rst]:
```suggestion
======================
```

https://travis-ci.org/esphome/esphome-docs/jobs/597550650#L239

File: components/sensor/xiaomi_cgg1.rst [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+    The ESPHome Xiaomi integration listens passively to packets the xiaomi device sends by itself.
+    ESPHome therefore has no impact on the battery life of the device.
+
+See Also
+--------
+
+- :doc:`/components/esp32_ble_tracker`
+- :doc:`/components/sensor/xiaomi_mijia`
+- :doc:`/components/sensor/index`
+- :apiref:`xiaomi_CGG1/xiaomi_CGG1.h`
```
Comment (OttoWinter) [inline components/sensor/xiaomi_cgg1.rst]:
```suggestion
- :apiref:`xiaomi_cgg1/xiaomi_cgg1.h`
```

File: components/sensor/xiaomi_cgg1.rst [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+  - **name** (**Required**, string): The name for the humidity sensor.
+  - **id** (*Optional*, :ref:`config-id`): Set the ID of this sensor for use in lambdas.
+  - All other options from :ref:`Sensor <config-sensor>`.
+
+
+Setting Up Devices
+------------------
+
+To set up Xiaomi CGG1 devices you first need to find their MAC Address so that ESPHome can
+identify them. So first, create a simple configuration without any ``xiaomi_CGG1`` entries like so:
```
Comment (OttoWinter) [inline components/sensor/xiaomi_cgg1.rst]:
```suggestion
identify them. So first, create a simple configuration without any ``xiaomi_cgg1`` entries like so:
```

File: components/sensor/xiaomi_cgg1.rst [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+it detects these sensors, it will automatically parse the BLE message print a message like this one:
+
+.. code::
+
+    Xiaomi CGG1 58:2D:34:10:7A:8C Got temperature=23.4°C, humidity=65.0%
+
+Note that it can sometimes take some time for the first BLE broadcast to be received. You can speed up
+the process by pressing the grey bluetooth button on the back of the device.
+
+Then just copy the address (``58:2D:34:10:7A:8C``) into a new ``sensor.xiaomi_CGG1`` platform entry like
```
Comment (OttoWinter) [inline components/sensor/xiaomi_cgg1.rst]:
```suggestion
Then just copy the address (``58:2D:34:10:7A:8C``) into a new ``sensor.xiaomi_cgg1`` platform entry like
```

**Review by OttoWinter — COMMENTED** [review]
Looks mostly good,

but you will need to create a minified version of the image (see contributing guide) in the images/ folder and link the document on the front page (index.rst, also contributing guide).

**Comment by Alex9779** [comment]
sorry for the things I missed, I read the guide but forgot about it...

**Comment by Alex9779** [comment]
closing in favor of #361

**Pull request closed without merging**